packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/take-profit.tsx

```diff
@@ -73,7 +68,9 @@ const TakeProfit = observer((props: TTakeProfitProps) => {
                 name='take_profit'
                 onChange={changeValue}
                 setCurrentFocus={setCurrentFocus}
-                tooltip_label={tooltip_text}
+                tooltip_label={
+                    <Localize i18n_default_text='When your profit reaches or exceeds this amount, your trade will be closed automatically.' />
+                }
```

Comment:
Unification of 'Take profit' text for all trade types everywhere including this one displayed in Trade params section.